brainiak/fcma/io.py

```diff
@@ -24,46 +31,49 @@
 
 logger = logging.getLogger(__name__)
 
-def read_activity_data(dir, file_extension, mask_file):
-    """ read data in NIfTI format and apply the spatial mask to them
+
+def read_activity_data(dir, file_extension, mask_file=None):
+    """ read data in NIfTI from a dir and apply the spatial mask to them
 
     Parameters
     ----------
     dir: str
         the path to all subject files
     file_extension: str
         the file extension, usually nii.gz or nii
-    mask_file: str
+    mask_file: str, default None
```

Comment:
You should document this as `Optional[str]`:
https://www.python.org/dev/peps/pep-0484/#union-types